searchlib/src/vespa/searchlib/tensor/hnsw_index.cpp

```diff
@@ -288,6 +288,16 @@ calc_distance_helper(const BoundDistanceFunction &df, vespalib::eval::TypedCells
     return df.calc(rhs);
 }
 
+void prefetch_docid(const DocVectorAccess& vectors, uint32_t docid)
+{
+    vectors.prefetch_docid(docid);
+}
+
+void prefetch_vector(const DocVectorAccess& vectors, uint32_t docid)
+{
+    vectors.prefetch_vector(docid);
+}
```

Comment:
Should these just be `vectors.prefetch_...` directly in the caller instead of going via forwarding freestanding functions?